nautobot/extras/tests/test_filters.py

```diff
@@ -1116,6 +1121,48 @@ def test_is_job_hook_receiver(self):
         self.assertEqual(self.filterset(params, self.queryset).qs.count(), 4)
 
 
+class JobQueueFilterSetTestCase(FilterTestCases.FilterTestCase):
```

Comment:
*Maybe* include `TenancyFilterTestCaseMixin`? It's a bit redundant with the `generic_filter_tests` now but it does cover the `tenant_group` filter that you're otherwise missing a test for here.